test/general/multiple_translation_units/header_umbrella.h

```diff
@@ -0,0 +1,53 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "support/test_config.h"
+
+#include <oneapi/dpl/algorithm>
+#include <oneapi/dpl/array>
+
+#if TEST_DPCPP_BACKEND_PRESENT
+#   include <oneapi/dpl/async>
+#endif
+
+#include <oneapi/dpl/cmath>
+#include <oneapi/dpl/complex>
+#include <oneapi/dpl/cstddef>
+#include <oneapi/dpl/cstring>
+
+#if TEST_DPCPP_BACKEND_PRESENT && __has_include(<oneapi/dpl/dynamic_selection>)
```

Comment:
In what situation would this test be compiled with a copy of oneDPL that does not include the `dynamic_selection` header? Same question for the `kernel_templates` header.

Reply:
I thought about using that test with older oneDPL versions, where these headers are not available, e.g. backward compatibility testing context. Perhaps, it is too much. I will remove these checks.

Reply:
Removed